WORKSPACE

```diff
@@ -10,6 +10,10 @@ load("//third_party/toolchains/gpu:cuda_configure.bzl", "cuda_configure")
 
 cuda_configure(name = "local_config_cuda")
 
+load("//tensorflow_io/core/kernels/gstpufs:workspace.bzl", "gstpufs_repositories")
+
+gstpufs_repositories()
```

Comment:
Those two lines are not needed. The `workspace.bzl` style in TF core are mostly comes from Bazel's pre-1.0 days when many features are not available in Bazel. So TF was forced to patch around with different `.bzl` files.

Now bazel is in 3.0+, just copy the content from `workspace.bzl` to here, remove workspace.bzl, and remove the above two lines will be fine:
```
    http_archive(
         name = "libmemcached",
         urls = [
             "https://storage.googleapis.com/mirror.tensorflow.org/launchpad.net/libmemcached/1.0/1.0.18/+download/libmemcached-1.0.18.tar.gz",
             "https://launchpad.net/libmemcached/1.0/1.0.18/+download/libmemcached-1.0.18.tar.gz",
         ],
         sha256 = "e22c0bb032fde08f53de9ffbc5a128233041d9f33b5de022c0978a2149885f82",
         strip_prefix = "libmemcached-1.0.18",
         build_file = "//third_party:libmemcached.BUILD",
     )
```